summary_workflows/identification/I2_benchmark/I2_benchmark_specification.md

```diff
@@ -52,6 +59,64 @@ Fields:
 This BED file contains single-nucleotide position of poly(A) sites identified from the orthogonal 3'end-seq dataset.  
 Fields are the same as in Format 1.
 	
+## Plots
+
+The results of this benchmark will be visualised in OpenEBench using the following plots:
+
+1. **bar plot** visualizing **Precision** of poly(A) site identification. Separate plots should be prepared for different values of distance threshold:
+
+- 10 nt
+- 50 nt
+- 100 nt
+
+**X axis** - benchmarked tool  
+**Y axis** - precision
+
+Ranking: The best performing tool is the one with the highest precision value.
+
+2. **bar plot** visualizing **Sensitivity** of poly(A) site identification. Separate plots should be prepared for different values of distance threshold:
+
+- 10 nt
+- 50 nt
+- 100 nt
+
+**X axis** - benchmarked tool  
+**Y axis** - sensitivity
+
+Ranking: The best performing tool is the one with the highest sensitivity value.
+
+3. **2D scatter plot** visualizing **TPR and FPR** of poly(A) site identification. Separate plots should be prepared for different values of distance threshold:
+
+- 10 nt
+- 50 nt
+- 100 nt
+
+**X axis** - FPR(d)  
+**Y axis** - TPR(d)  
+where _d - distance threshold_
+
+Ranking: The best performing tool is the one with the highest TPR combined with lowest FPR (top left part of the plot) and the worst performing tool is the one with the lowest TPR combined with highest FPR (bottom left part of the plot. The plot should be divided into diagonal quartiles based on the distance from optimal performance. Alternatively, if the plot is divided into square quartiles, the following ranking order should be applied: top-left, top-right, bottom-left, bottom-right.
+
+4. **2D line plot** visualising **TPR and FPR** in the form of [ROC curve][roc-curve], i.e. TPR and FPR calculated for a range of distance thresholds are plotted together.
+
+**X axis** - FPR(d)  
+**Y axis** - TPR(d)  
+where _d - distance threshold_
+
+Ranking: For each tool, the area under the curve is calculated. The best performing tool is the one with the highest AUC.
+
+Note: 2D line plot is not supported in OpenEBench yet. If it's not implemented, the data should be visualised outside of OpenEBench.
+
+Optional plots:
+
+5. **bar plot** visualizing **AUC** of ROC plot with single AUC value for each tool.
+
+**X axis** - benchmarked tool  
+**Y axis** - AUC
+
+Ranking: The best performing tool is the one with the highest AUC value.
+
```

Comment:
Since #5 can be addressed with the current functionality of OpenEBench, I would use this instead of #4